### PR TITLE
Add Sublime and Grunt to list of integrations

### PR DESCRIPTION
**Pull request by Kristinita**
### 1. Summary

I add remark integration for Sublime Text and Grunt.

#### 2.1. Grunt

grunt-remark [**poorly documented**](https://www.npmjs.com/package/grunt-remark), but successfully works for me with plugins, presets and configurations.

#### 2.2. Sublime Text

See my [**pull request**](https://github.com/j616/SublimeLinter-contrib-remark-lint/pull/1), it fixes error and add support for any plugin, preset and configuration.

Thanks.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -320,6 +320,10 @@ To run **remark**, optionally with **remark-lint** from **Gulp**, use
 
 To check markdown documents editing with Vim, use [**ale**][ale].
 
+For [**Sublime Text**](https://www.sublimetext.com) use [**SublimeLinter-contrib-remark-lint**](https://packagecontrol.io/packages/SublimeLinter-contrib-remark-lint) — part of [**SublimeLinter**](http://www.sublimelinter.com/en/stable/) framework.
+
+[**Grunt**](https://gruntjs.com/) users can use [**grunt-remark**](https://www.npmjs.com/package/grunt-remark).
```
Comment (ChristianMurphy) [inline readme.md]: Grunt is more a build tool than an editor.
Maybe add in a new section for build too integrations?

```suggestion
## Build Tool Integrations

[**Grunt**](https://gruntjs.com/) users can use [**grunt-remark**](https://www.npmjs.com/package/grunt-remark).
```

Comment (Kristinita) [reply readme.md]: @ChristianMurphy, personally, I would make a common table without extra words; example:

| [**Sublime Text**](https://www.sublimetext.com) | [**SublimeLinter-contrib-remark**](https://packagecontrol.io/packages/SublimeLinter-contrib-remark-lint) |
|-------------------------------------------------|----------------------------------------------------------------------------------------------------------|
| [**Grunt**](https://gruntjs.com/)               | [**grunt-remark**](https://github.com/remarkjs/grunt-remark)                                             |

But it does not matter to me; do as you wish.

Thanks.

Comment (ChristianMurphy) [reply readme.md]: My thought would be to keep the format as is to be consistent with other remark/unified plugins.
@Murderlon @wooorm thoughts? :thought_balloon:

Comment (wooorm) [reply readme.md]: @Kristinita Also nice!

@ChristianMurphy Hmm, Gulp is in the first section, so I think splitting this up makes more sense in a separate PR! I’d personally go with a list of things, just like most other lists of plugins/related projects, e.g.:

```markdown
## Integrations

*   [linter-markdown](#) ([Atom](#)) — use remark-lint from Atom
*   [linter-remark](#) ([Atom](#)) — use all of remark from Atom
*   [SublimeLinter-contrib-remark-lint](#) ([Sublime](#)) — use remark-lint from Sublime Text
*   [ale](#) ([Vim](#)) — use remark-lint from Vim
*   [gulp-remark](#) ([Gulp](#)) — use all of remark with Gulp
*   [grunt-remark](#) ([Grunt](#)) — use all of remark with Grunt

We’re interested in more integrations. Let us know if we can help.
```

»

## Integrations

*   [linter-markdown](#) ([Atom](#)) — use remark-lint from Atom
*   [linter-remark](#) ([Atom](#)) — use all of remark from Atom
*   [SublimeLinter-contrib-remark-lint](#) ([Sublime](#)) — use remark-lint from Sublime Text
*   [ale](#) ([Vim](#)) — use remark-lint from Vim
*   [gulp-remark](#) ([Gulp](#)) — use all of remark with Gulp
*   [grunt-remark](#) ([Grunt](#)) — use all of remark with Grunt

We’re interested in more integrations. Let us know if we can help.

**Pull request merged**